content/en/containers/guide/operator-eks-addon.md

```diff
@@ -0,0 +1,152 @@
+       clusterName: <CLUSTER_NAME>
+       registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com/datadog
+       credentials:
+         apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+         appSecret:
+           secretName: datadog-secret
+           keyName: app-key
+     features:
+       apm:
+         enabled: true
+       logCollection:
+         enabled: true
+   ```
+   This agent instance configuration pulls the Datadog agent image from an AWS Marketplace hosted ECR repository which also contains the image for the Datadog Operator Amazon EKS add-on. Should you require alternatives, kindly edit the 'global.registry' entry in the manifest above.
+
+   For all configuration options, see the [Operator configuration spec][6].
+
+   **Note:** If access to IMDS v1 is blocked on the node, Agent will not be able to resolve the cluster name and certain features for example [Orchestrator Explorer][6] will not work. Hence, it is recommended to add `spec.global.ClusterName` in the `DatadogAgent` manifest. See this [comment][8] on how to configure Agent to request metadata using IMDS v2.
+
+4. Deploy the Datadog Agent:
+   ```bash
+   kubectl apply -f /path/to/your/datadog-agent.yaml
+   ```
+
+
+## Uninstalling Operator
+
+If you want to uninstall the Agent and Operator, first delete the `DatadogAgent` custom resource:
+
+  ```bash
+  kubectl delete datadogagents.datadoghq.com datadog
+  ```
+
+Confirm all Agent resources are deleted and proceed with add-on uninstallation:
+
+{{< tabs >}}
+{{% tab "Console" %}}
+
+* Go to the EKS cluster in the AWS console.
+* Go to the add-on tab and select *Datadog Operator* addon.
+* Click Remove and the confirmation prompt.
+
+{{% /tab %}}
+{{% tab "CLI" %}}
+
+To delete the add-on, run:
+  ```bash
+  aws eks delete-addon --addon-name datadog_operator --region <AWS_REGION> --cluster-name <CLUSTER_NAME>
+  ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+ **Note:** If Operator add-on is uninstalled before deleting the `DatadogAgent` custom resource, agents will continue to run on the cluster. Deleting namespace will get stuck since the `DatadogAgent` can not be finalized without a running Operator. See this Github [issue][9] for a workaround.
+
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://aws.amazon.com/marketplace/pp/prodview-wedp6r37fkufe
+[2]: /getting_started/containers/datadog_operator
+[3]: https://github.com/DataDog/helm-charts/blob/operator-eks-addon/charts/operator-eks-addon/aws_mp_configuration.schema.json
+[4]: https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator
+[5]: https://app.datadoghq.com/account/settings#api
+[6]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
+[7]: https://docs.datadoghq.com/infrastructure/containers/orchestrator_explorer/?tab=datadogoperator
+[8]: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L407-L416
```

Comment:
I'm a bit worried that this YAML file may change over time. Consider using a permalink instead (https://github.com/DataDog/datadog-agent/blob/4896a45f586f74de1da2e985f98988f0181afc36/pkg/config/config_template.yaml#L407-L416).

Reply:
Updated, thanks for the suggestion!